tracer/src/Datadog.Trace/SpanContext.cs

```diff
@@ -299,12 +314,14 @@ IEnumerator IEnumerable.GetEnumerator()
         internal static class Keys
         {
             private const string Prefix = "__DistributedKey-";
+
             public const string TraceId = $"{Prefix}TraceId";
             public const string ParentId = $"{Prefix}ParentId";
             public const string SamplingPriority = $"{Prefix}SamplingPriority";
             public const string Origin = $"{Prefix}Origin";
             public const string RawTraceId = $"{Prefix}RawTraceId";
             public const string RawSpanId = $"{Prefix}RawSpanId";
+            public const string DatadogTags = $"{Prefix}DatadogTags";
```

Comment:
Should we rename this field to `PropagatedTags` if that's the new name for it? Similarly, should we change the value to `$"{Prefix}DatadogTags";`?

Reply:
Good catch, thanks!